packages/components-library/tests/demo-data/filterMocks.ts

```diff
@@ -0,0 +1,181 @@
+export const fruitOptions = [
+  { text: 'Orange', value: 'orange' },
+  { text: 'Apple', value: 'apple' },
+  { text: 'Pineapple', value: 'pineapple' },
+  { text: 'Grape', value: 'grape' },
+  { text: 'Melon', value: 'melon' },
+  { text: 'Kiwi', value: 'kiwi' },
+  { text: 'Blueberry', value: 'blueberry' },
+  { text: 'Pear', value: 'pear' },
+  { text: 'Apricot', value: 'apricot' },
+  { text: 'Mango', value: 'mango' },
+  { text: 'Breadfruit', value: 'breadfruit' },
+  { text: 'Cherry', value: 'cherry' },
+  { text: 'Cranberry', value: 'cranberry' },
+  { text: 'Durian', value: 'durian' },
+  { text: 'Guava', value: 'guava' },
+  { text: 'Papaya', value: 'papaya' }
+]
+
+export const fruitOptionsFunction = () => Promise.resolve(fruitOptions)
```

Comment:
Please don't add type to variable names. Just rename to `fruitOptions`?

Reply:
In the general case a would agree, but as this variable is use in the example ( that does not support typescript) it is not clear that fruitOptions is a function ( fruitOptions is a list of fruit). Adding the Function to the name help with both the test and example readability in this case i would argue